src/components/Table/GridTable.stories.tsx

```diff
@@ -199,6 +203,323 @@ export function NestedRows() {
+        <div data-card-open="level3">
+          <div data-level="level1">
+            <div data-level="level2">
+              <div data-level="level3" />
+            </div>
+          </div>
+        </div>
+        <div data-card="level3" css={Css.display("contents").$}>
+          <div data-level="level1" css={Css.pl1.$}>
+            <div data-level="level2" css={Css.pl1.$}>
+              <div data-level="level3" css={Css.pl1.$}>
+                Task 2
+              </div>
+            </div>
+          </div>
+          <div data-level="level3">Task 2</div>
+          <div data-level="level1" css={Css.pr1.$}>
+            <div data-level="level2" css={Css.pr1.$}>
+              <div data-level="level3" css={Css.pr1.$}>
+                Task 2
+              </div>
+            </div>
+          </div>
+        </div>
+        <div data-card-close="level3">
+          <div data-level="level1">
+            <div data-level="level2">
+              <div data-level="level3" />
+            </div>
+          </div>
+        </div>
+        <div data-card-close="level2">
+          <div data-level="level1">
+            <div data-level="level2" />
+          </div>
+        </div>
+
+        {/*Child has closed, so spacer level1.*/}
+        <div data-spacer="true">
+          <div data-level="level1" />
+        </div>
+
+        {/* Child row, could be "last grand-child" as well as "last child" */}
+        <div data-card-open="level2">
+          <div data-level="level1">
+            <div data-level="level2" />
+          </div>
+        </div>
+        <div data-card="level2" css={Css.display("contents").$}>
+          <div data-level="level1" css={Css.pl1.$}>
+            <div data-level="level2" css={Css.pl1.$}>
+              Group 3
+            </div>
+          </div>
+          <div data-level="level2">Group 3</div>
+          <div data-level="level1" css={Css.pr1.$}>
+            <div data-level="level2" css={Css.pr1.$}>
+              Group 3
+            </div>
+          </div>
+        </div>
+        <div data-card-close="level2">
+          <div data-level="level1">
+            <div data-level="level2" />
+          </div>
+        </div>
+        <div data-card-close="level1">
+          <div data-level="level1" />
+        </div>
+
+        {/*Child has closed, so spacer level0.*/}
+        <div data-spacer="true" />
+
+        <div data-card-open="level1">
+          <div data-level="level1" />
+        </div>
+        <div data-card="level1" css={Css.display("contents").$}>
+          <div data-level="level1" css={Css.pl1.$}>
+            Milestone 2
+          </div>
+          <div data-level="level1">Milestone 2</div>
+          <div data-level="level1" css={Css.pr1.$}>
+            Milestone 2
+          </div>
+        </div>
+        <div data-card-close="level1">
+          <div data-level="level1" />
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export function OneOffInlineTable() {
```

Comment:
Not related to the nested cards, but thought I'd copy/paste this over from internal-frontend where it's our "you don't really need gridtable to do this tiny/one-off layout" snippet.

Reply:
Makes me feel like we should provide a SimpleTable component or something... Maybe not? I dunno. Just feels like this could easily get inconsistent if we start having these little one-off tables with localized styles everywhere.

Reply:
Yeah. I balance 100% agreement with "...but that is what mocks just do..." i.e. designers have the expectation of being able to bespoke layout a mini-table.

Maybe we could nudge them towards consistency with something like a `SimpleTable` + 2-3 out-of-the-box styles, i.e. like grid tables style `const`s, maybe with links to previous/standard figma mocks for each style (i.e. probably feature-specific figmas b/c we're extracting these as we go), so that when a new mock goes by with a "not grid table" table (as in it's not a sorting/filtering/data table and instead just some mini-info laid out nicely), we could link to "this mock did that, and it looked like x, can we just use that style?".